hook/mutate_pod.go

```diff
@@ -161,3 +171,29 @@ func (m podMutator) requestedCapacity(ctx context.Context, pod *corev1.Pod, targ
 	}
 	return total, nil
 }
+
+func (m podMutator) requestedEphemeralCapacity(pod *corev1.Pod) (int64, error) {
+	var total int64
+	sizeAttribute := "topolvm.cybozu.com/size"
```

Comment:
As noted, this should be defined externally as a constant.

Reply:
What do you mean by "externally"? Not sure exactly where you are suggesting this should go.

Reply:
Please define a constant for `topolvm.cybozu.com/size` literal and use it here and [there](https://github.com/cybozu-go/topolvm/pull/93#discussion_r373932673).

Reply:
Done.